sklearn/isotonic.py

```diff
@@ -252,7 +278,7 @@ def _build_f(self, X, y):
             # single y, constant prediction
             self.f_ = lambda x: y.repeat(x.shape)
         else:
-            self.f_ = interpolate.interp1d(X, y, kind='slinear',
+            self.f_ = interpolate.interp1d(X, y, kind='linear',
```

Comment:
No, I just confirmed that indeed linear interpolation is much faster. But I'm actually encouraging further look into why interpolation method was changed in the first place because I'm pretty sure it was related to several isotonic regression bugs reported in version 0.15.0. Just as background, Isotonic Regression used to use linear interpolation up to 0.15.0, then it was changed to slinear in 0.16.0 and now it might be changed again back to linear by this PR.